metabase/metabase
# Make ExternalLink accessible

Epic: https://github.com/metabase/metabase/issues/19696

How to test:
- Open storybook `yarn storybook`
- Navigate to `ExternalLink`
- Make sure it receives focus on tab
